IDE/Espressif/ESP-IDF/libs/CMakeLists.txt

```diff
@@ -38,10 +38,14 @@ if(IS_DIRECTORY ${IDF_PATH}/components/cryptoauthlib)
 endif()
 
 set(COMPONENT_SRCEXCLUDE
-    "wolfcrypt/src/aes_asm.S"
```

Comment:
Did you intend to remove `wolfcrypt/src/aes_asm.S`?

Reply:
I intended to remove it. It isn't pulled into the build without being listed here.